### PR TITLE
Codeception test setup

### PR DESCRIPTION
**Pull request by IvanGrginovInf**
Different target branch for https://github.com/infinum/eightshift-libs/pull/86

### CODE REVIEW

**Review by dingo-d — APPROVED** [review]

**Review by iruzevic — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
+  "prefer-stable": true,
+  "config": {
+    "sort-packages": true,
+    "optimize-autoloader": true,
+    "process-timeout": 2000
+  },
+  "scripts": {
+    "analyze": "@php ./vendor/bin/phpstan analyze",
+    "standards:check": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs",
 		"standards:fix": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf",
```
Comment (dingo-d) [inline composer.json:66]: Why is this line indented weirdly?

File: composer.json [hunk 2]
```diff
@@ -1,68 +1,70 @@
...
+    "szepeviktor/phpstan-wordpress": "^0.7",
+    "wp-cli/wp-cli": "^2.4"
+  },
+  "autoload": {
+    "psr-4": {
+      "EightshiftLibs\\": "src/",
+      "EightshiftBoilerplate\\": "src/"
+    }
+  },
+  "minimum-stability": "dev",
```
Comment (dingo-d) [inline composer.json]: `autoload-dev` should be added for tests as well, something like

```json
"autoload-dev": {
    "psr-4": {
        "Tests\\": "tests/",
    }
},
```

**Review by dingo-d — CHANGES_REQUESTED** [review]
Left a comment

File: composer.json (hunk 1 above)
Comment (IvanGrginovInf) [reply composer.json:66]: I fixed this already (tabs instead of spaces) 🙈 will fix

**Review by iruzevic — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Pull request merged**